firefox-ios/Client/Frontend/Browser/Tabs/Views/EmptyPrivateTabsView.swift

```diff
@@ -67,11 +67,17 @@ class EmptyPrivateTabsView: UIView {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    private func configureLearnMoreButton() {
+        let viewModel = LinkButtonViewModel(title: .PrivateBrowsingLearnMore,
+                                            a11yIdentifier: "a11yLearnMoreButton",
```

Comment:
A small change request: Let's move the hardcoded: `a11yIdentifier` to `AccessibilityIdentifiers.TabTray`.

Reply:
Okay, sounds good. Will update shortly 👍